docs/Common/Structs/index.md

```diff
@@ -1179,3 +1182,40 @@ There are no defined parameters for this struct
 |:---|:---|:--------|:---------|:----------|
 |location|Common.Grid|true|||
 |state|Common.WindowState|true|||
+
+### DoorStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|Common.Grid|true||Describes the status of a location of a door.|
+|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of a door.|
+
+### GateStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|Common.Grid|true||Describes the status of location of trunk/hood/etc.|
```

Comment:
Please add reference links for Grid, DoorStatusType

Reply:
the same as above, `Grid` is not worth adding.
`DoorStatusType` is already there - line 1198

`|status|[Common.DoorStatusType](../enums/#doorstatustype)|true||Describes the status of trunk/hood/etc.|`

Reply:
Same comment as above for `Enums` case.

Reply:
@atiwari9, please see the links to enums updated per your request in 82d254e